blocks/search-results-list-block/features/search-results-list/_children/search-result.test.jsx

```diff
@@ -14,7 +14,7 @@ jest.mock('fusion:properties', () => (jest.fn(() => ({
 describe('The search results', () => {
   describe('renders one list item correctly', () => {
     const element = oneListItem.data[0];
-    if (element?.promo_items?.basic) {
+    if (element?.['promo_items']?.basic) {
```

Comment:
You can remove this style preference as well unless it was deliberate

Reply:
It doesn't show here, but `element?.promo_items?.basic` required an eslint exception

Reply:
weird that it would cause an eslint error. I'm seeing     `if (element?.['promo_items']?.basic) {` cause an exception when I run `rm -rf node_modules/ && npm i && npm run lint`